### DISCUSSION TITLE
Update-2023-08-03 Force Fields

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#238] [Announcements]
Added a new `time-varying-force-field` class which uses noise. Two results below, both using the new field to influence the mass at the end of the Spirograph spring (red vector). In this way, the curve never reaches an equilibrium state. The second example has ground collisions enabled. 

![Screen Shot 2023-08-03 at 11 05 42 AM](https://github.com/kaveh808/kons-9/assets/103860793/91ace621-180f-4395-9104-6067ade15754)

![Screen Shot 2023-08-03 at 11 07 40 AM](https://github.com/kaveh808/kons-9/assets/103860793/ca806087-7ab9-4d04-8f15-cf7f2c0d81f8)

**Comment by aykaramba** [comment]
Nice!
